chainermn/communicators/_base.py

```diff
@@ -20,6 +20,9 @@ def rank(self):
     def size(self):
         return self.mpi_comm.size
 
+    def split(self, color, key):
```

Comment:
Maybe we want an explanation on the parameter 'color'

Reply:
Added some detailed comments.